jujugui/static/gui/src/app/components/inspector/test-inspector.js

```diff
@@ -73,7 +75,34 @@ describe('Inspector', function() {
     var output = shallowRenderer.getRenderOutput();
     assert.deepEqual(output.props.children[1].props.children,
         <juju.components.UnitList
-          units={['units']}/>);
+          serviceId="demo"
+          units={['units']}
+          changeState={undefined} />);
```

Comment:
we should pass a stub in on line 72 and then check for it here just to make sure it is getting passed through